packages/babel-plugin-proposal-decorators/test/fixtures/2021-12-classes--to-es2015/replacement-static-installed-on-correct-class/exec.js

```diff
@@ -0,0 +1,33 @@
+let hasX, hasM, OriginalFoo;
```

Comment:
@pzuraq Can you check if this test looks correct?

Reply:
Yes this does look correct! Thanks for updating it